linux/just_files/docker_functions.bsh

```diff
@@ -754,6 +793,125 @@ function docker_compose_volumes()
   done
 }
 
+#**
+# :Arguments: * ``$1`` - Name of service of interest
+#             * [``$2``] - Prefix to volume names
+# :Input: **stdin** - Yarped docker compose yaml file
+# :Output: * ``DOCKER_VOLUME_SOURCES`` - List of sources
+#          * ``DOCKER_VOLUME_TARGETS`` - Corresponding list of targets
+#          * ``DOCKER_VOLUME_FLAGS`` - Corresponding list of flags. Can be a mix of short and long format flags. Needs more parsing before this mix is useful.
+#          * ``DOCKER_VOLUME_FORMATS`` - Corresponding list specifying either "long" or "short" as the format of each volume
+#
+# Get volume lists from ``DOCKER_VOLUME_LINES``
+#
+# .. rubric:: Usage
+#
+# .. code-block:: bash
+#
+#    vsi::docker::get_compose_volumes nb < <(docker compose config | yarp)
+#    docker_compose_volumes
```

Comment:
why are you saying call docker_compose_volumes? isn' that the deprecated version of get_compose_volumes